op-dispute-mon/mon/service.go

```diff
@@ -129,6 +129,14 @@ func (s *Service) initGameCallerCreator() {
 	s.game = extract.NewGameCallerCreator(s.metrics, s.l1Caller)
 }
 
+func (s *Service) asOutputRollupClients() []extract.OutputRollupClient {
```

Comment:
Given this wound up as an instance method I'd probably make it:
```suggestion
func (s *Service) outputRollupClients() []extract.OutputRollupClient {
```

which is a pretty good option actually.  What I was originally thinking was:
```
func asOutputRollupClients(clients []*rollup.Client) []extract.OutputRollupClients {
```

Reply:
Oops, fixed the name. I can still move it to a pure function if you like though (named with `as...`, no strong opinion on this.